jquery/exercise8/js/slideshow.js

```diff
@@ -0,0 +1,38 @@
+function SlideShow(slideShow ,header ,list) {
+  this.$slideShowElement = slideShow;
+  this.$header = header;
+  this.$list = list;
+  this.$element;
+  this.$nextElement;
+};
+
+SlideShow.prototype.bindEvents = function() {
```

Comment:
which event is binding in this function??
if you are not binding any event
then why this is bindEvents?